isis/src/control/objs/BundleUtilities/BundleMeasure.h

```diff
@@ -94,6 +102,13 @@ namespace Isis {
                                                      bundle control measure **/
       QSharedPointer<BundleImage> m_parentBundleImage; /**< Parent image of this bundle control measure **/
       QSharedPointer<BundleObservation> m_parentObservation; /**< Parent bundle observation **/
+      
+      LinearAlgebra::Matrix *m_imagePartial;    /**< Image partials associated with the control measure **/
+      LinearAlgebra::Matrix *m_point3DPartial;  /**< Point partials associated with the control measure **/
+      LinearAlgebra::Matrix *m_targetPartial;   /**< Target partials associated with the control measure **/
+
+      // Test statistic used for determining gross outliers
+      LinearAlgebra::Vector m_grossOutlierTestStatistic;
```

Comment:
This should either be the entire covariance matrix for the measure or two separate values (sampleTestStatistic and lineTestStatistic). It doesn't make sense to store this in a vector.